src/halite/features/logging/common.cr

```diff
@@ -17,7 +17,7 @@ class Halite::Logging
   class Common < Abstract
     def request(request)
       message = String.build do |io|
-        io << "> | request  | " << current_timestamp << colorful_method(request.verb)
```

Comment:
Thanks for this change, now I don't even need to override anything in the logs, looks very nice!